docs/files/index.rst

```diff
@@ -0,0 +1,16 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Handling Files
+==============
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
What do you think about having `maxdepth` set to `2` instead so we can see the sub-sections and more easily navigate?